src/Admin.php

```diff
@@ -30,6 +30,38 @@ public static function init() {
     if (get_option('_' . Plugin::L10N . '_enable_auto_sale_category_assignment') === 'yes') {
       add_action('woocommerce_update_product', __NAMESPACE__ . '\WooCommerce::woocommerce_update_product');
     }
+
+    // Allow ajax requests to specified functions.
+    add_action('wp_ajax_is_existing_gtin', __NAMESPACE__ . '\WooCommerce::wp_ajax_is_existing_gtin');
+
+    // Enqueues admin plugin scripts.
+    add_action('admin_enqueue_scripts', __CLASS__ . '::admin_enqueue_scripts');
+
+    Settings::init();
+  }
+
+  /**
+   * Enqueues admin plugin scripts.
+   *
+   * @implements admin_enqueue_scripts
+   */
+  public static function admin_enqueue_scripts($hook) {
+    $git_version = Plugin::getGitVersion();
+    global $post;
+    if ($hook == 'post-new.php' || $hook == 'post.php') {
```

Comment:
```suggestion
    if ($hook === 'post-new.php' || $hook === 'post.php') {
```

Reply:
@moritzbappert Can you apply this change and/or grant us write access to your fork, please?

Reply:
@sun @LucaPipolo I have just applied the suggested changes!